extensions/cms/src/apiWrapper.ts

```diff
@@ -269,20 +282,17 @@ export class ApiWrapper {
 				authTypeChanged: true
 			}
 		};
-		return this.openConnectionDialog(['MSSQL-CMS'], initialProfile, undefined).then((connection) => {
+		return this.openConnectionDialog([cmsProvider], initialProfile, { saveConnection: false }).then(async (connection) => {
```

Comment:
what does `saveConnection: false` do here?  It seems like we are saving the connection the CMS view, so the parameter seems a little unintuitive.

Reply:
The CMS server always gets saved locally in a different setting. This is for not saving into the OE and `Saved Connections` since if it explicitly set to false, they will be saved.